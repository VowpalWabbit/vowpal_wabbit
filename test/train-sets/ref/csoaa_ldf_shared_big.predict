0
0
0.316015
0.481197
0.889338
0.641173
0.641173
1.450128
1.452039
1.529907
1.781868
0.785337
0.785337
1.591799
1.806777
2.009719
2.286706
0.852653
0.496039
1.563403
1.973132
2.308722
2.633690
