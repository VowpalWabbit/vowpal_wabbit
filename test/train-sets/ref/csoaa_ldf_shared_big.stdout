[warning] 'shared' is not a good float, replacing with 0
[warning] 'shared' is not a good float, replacing with 0
[warning] 'shared' is not a good float, replacing with 0
[warning] 'shared' is not a good float, replacing with 0
